reference/tax_settings.v3.yml

```
@@ -178,6 +190,18 @@ components:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, geolocation will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store.
+        guest_shopper_tax_zone_id:
+          default: 1
+          type: integer
+          description: Describes which tax zone is used for guest shoppers when requiring storefront tax estimation. When applicable, the shopper's geolocation data will be used in favour of the guest shopper tax zone.
+        store_tax_zone_id:
+          default: 1
+          type: integer
+          description: This setting applies only if a merchant enters tax-inclusive prices. Describes which tax zone to use for store tax subtraction.
```

```suggestion
          description: This setting applies only if a merchant enters tax-inclusive prices and store tax subtraction is enabled. Describes which tax zone to use for store tax subtraction.
```

Thanks for the suggestion here @bc-msung, great call. Good to, in context, identify when this setting is relevant, guiding the user towards using/not using it as relevant.

---

👍